vendor/src/github.com/microsoft/hcsshim/createprocess.go

```diff
@@ -43,7 +86,7 @@ func CreateProcessInComputeSystem(id string, params CreateProcessParams) (proces
 	if err != nil {
 		err = fmt.Errorf(title+" - Failed conversion of id %s to pointer %s", id, err)
 		logrus.Error(err)
-		return 0, err
+		return
```

Comment:
@tiborvass This pattern is used in quite a few places, both in HCSShim and I've seen it in docker itself. Could you submit an issue against http://github.com/Microsoft/HCSShim and I'll go through and update them if that's what you would prefer